noises.md

```diff
@@ -194,3 +238,4 @@ https://www.youtube.com/watch?v=31zzMb3U0iY
 
 ### Loogie
 - Start by making the Hard H sound, then pull your lips back quickly
+custom glue-up
```

Comment:
```suggestion
```

Reply:
<img width="771" alt="Screen Shot 2019-03-19 at 1 37 07 AM" src="https://user-images.githubusercontent.com/2823112/54591492-93f39d00-49e7-11e9-86b6-804d1829a81b.png">

weird perms issues so can't fix this nit thru GitHub UI

Reply:
maybe b/c `master` is protected /shrug